### ISSUE TITLE
Does pgRouting 2.1 really require boost 1.55 or above?

### ISSUE THREAD

**Issue by robe2** [pgRouting/pgrouting#465]
I know I tried it on lower before and it worked, but I think I only went as far down as 1.53 or so.  I only upgraded just to be safe.

I asked yum.postgresql.org folks why they are shipping pgRouting 2.0.1 for 9.5 instead of pgRouting 2.1.0 and this is what they said:

Repeated from for completeness
http://www.postgresql.org/message-id/1453465521.24314.507.camel@gunduz.org

```
On Fri, 2016-01-22 at 02:08 -0500, Regina Obe wrote:
> Is there any reason you guys decided to package pgRouting 2.0.1
> instead of
> the newer pgRouting 2.1.0 with PostgreSQL 9.5.
> If not would it be a bother to replace (or add pgRouting 2.1).

2.1.0's boost requirement is >= 1.55, which is a great barrier for
older distros.

Regards,
-- 
Devrim GÜNDÜZ
Principal Systems Engineer @ EnterpriseDB: http://www.enterprisedb.com
PostgreSQL Danışmanı/Consultant, Red Hat Certified Engineer
Twitter: @DevrimGunduz , @DevrimGunduzTR
```

**Comment by cvvergara** [comment]
@robe2 
I don't know, I will have to download lower distributions to check what is the lowest...
For the moment, can you ask them what would be acceptable? So I can start from there?

**Comment by robe2** [comment]
I guess we really got to test this out.  I found out both apt.postgresql.org and debian are carrying just pgrouting 2.0.  http://irclogs.geoapt.com/postgis/%23postgis.2016-02-05.log

https://packages.qa.debian.org/p/pgrouting.html

When I installed pgrouting on my Ubuntu 14.04 via apt.postgresql.org, I got boost 1.54.  So perhaps the reason debian is only carrying 2.0 is for the same reason, which then has a trickle effect to Ubuntu.

Regarding the other item above.  I did discover setting up a CentOS 7  that I do get pgRouting 2.1 because CentOS 7 comes with a newer boost.
